docs/40_node-operation/10_getting-started/10_hardware-requirements.md

```diff
@@ -2,3 +2,47 @@
 title: Hardware Requirements
 sidebar_class_name: sidebarhidden
 ---
+
+The hardware requirements for each type of EOS node can vary depending on factors such as network size, transaction volume, and the number of concurrent users. However, here are some general guidelines for the hardware requirements of different EOS node types:
+
+## API Node without Block Log
+
+The minimum, starting point, for an API node, without maintaining block logs, is the following:
+
+- CPU: 2 cores >= 3.7 Ghz
+- RAM: 16 GB RAM and 16 GB swap
+- HDD: 1024 GB
+- Internet network link: 100Mb/s
+
+> ⚠ Be aware, the above configuration will face problems if suddenly RAM usage on the network increased over that 32GB mark. Also this setup is 50/50 RAM/swap and occasionally will respond slow on some requests due to swapping.
```

Comment:
Be aware, the above configuration will face -> Be aware that the above configuration could face

increased over -> increases over

Also -> Also, this